test/e2e/dra/dra.go

```diff
@@ -51,30 +52,34 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 	ginkgo.Context("kubelet", func() {
 		var driver = NewDriver(f, 1, 1 /* nodes */) // All tests get their own driver instance.
 
-		ginkgo.Context("", func() {
+		ginkgo.When(
+			"external claim is used in the 'wait for first consumer' allocation mode",
```

Comment:
This makes the overall test name very long. "with delayed allocation" should be enough.